cmd/kube-apiserver/app/options/options.go

```diff
@@ -45,7 +45,9 @@ type ServerRunOptions struct {
 	Etcd                    *genericoptions.EtcdOptions
 	SecureServing           *genericoptions.SecureServingOptions
 	InsecureServing         *kubeoptions.InsecureServingOptions
-	Audit                   *genericoptions.AuditLogOptions
+	Audit                   *genericoptions.AuditOptions
+	AuditLog                *genericoptions.AuditLogOptions
+	AuditWebhook            *genericoptions.AuditWebhookOptions
```

Comment:
nit: maybe the plugin options should just be nested in the general audit options?